src/mscorlib/shared/System/Boolean.cs

```diff
@@ -102,8 +102,7 @@ public bool TryFormat(Span<char> destination, out int charsWritten)
 
             if (s.Length <= destination.Length)
```

Comment:
Why not remove this branch entirely and optimistically call TryCopyTo, branching on its return statement? TryCopyTo will get inlined into its caller, and I confirmed through my test application that all of the branching collapses into a single statement. I think you'll get better code gen here as well.

Reply:
Yeah, that's better.

Reply:
Check the disassembly manually just to confirm, of course. :)